### DISCUSSION TITLE
Version check

### DISCUSSION THREAD

**Discussion by desertking** [ruffle-rs/ruffle#854] [General]
Hello,

how can i check which ruffle version is running? Is there a console-command i can enter in chrome?

**Comment by SoftCreatR** [comment]
There are two "versions":

1. The version itself (which can be queried via `window.RufflePlayer.version` when installed as described), which is 0.1.0 atm
2. The Name, which currently refers to the nightly build (e.g. `nightly 2021-03-01`)

However, depending on what you need, the version alone may not meet your requirements, because it currently won't help you to determine, if a new (nightly) version has been released, or not.

The problem is, that the Name isn't accessible, from what I can see. Maybe worth opening an issue or PR for that.

**Reply by desertking** [reply]
Hello,

back then ruffle didn't show the current version it's running on my own website, which it now does, even via right-click. I simply wanted to know if the cached version is really updated now to the new version and which version is running which was not possible at that time.

**Reply by SoftCreatR** [reply]
That's somewhat correct. The `version()` function has been added on 5 Sep 2020 via 4163ea7975d22ca86a7501e544f3536cba28137d
